configure.ac

```diff
@@ -2035,8 +2027,6 @@ AC_CHECK_FUNCS([tmpfile asprintf vasprintf waitpid])
 ## ----------------------------------------------------------------------
 ## Check compiler characteristics
 ##
-AC_C_CONST
```

Comment:
const is now defined in the C standard, so this archaic holdover is no longer needed.